packages/components/link/utils.ts

```diff
@@ -137,3 +139,39 @@ export const removeSourceUrl = (
     ? linkUrl.pathname + linkUrl.search + linkUrl.hash
     : link;
 };
+
+/**
+ * Checks if the provided link is an external Url.
+ *
+ * @param link - The link Url.
+ *
+ * @returns True if the link is an external Url.
+ */
+export const isExternalUrl = (link: string) => {
+  return (
+    link.startsWith("http") ||
+    link.startsWith("mailto") ||
+    link.startsWith("tel") ||
+    link.startsWith("sms")
```

Comment:
Hi @nicholasio thank you so much for taking the time and open this PR. Much appreciate it!

While looking at the changes was thinking if we maybe should approach it the other way around. Meaning, instead of checking that the url starts with a certain protocol, check if the url does not start with `/`?

Reply:
That's a good idea. I remember that we used links like `whatsapp://send?text=xxx` for the social media apps.

But instead of checking for `/`, why don't we check that there is a protocol?

We could use either a regexp or `new URL` for that:

```js
const isExternal = (link) => {
  try {
    new URL(link);
    return true;
  } catch (e) {
    return false;
  }
};
```

```js
const isExternal = (link) => /\w+:\/\//.test(link);
```

Reply:
Uh! @luisherranz that's so much better. Love it! 😃 I think the `URL` try/catch will do, but as you said either works. 👍

Reply:
Definitely makes more sense to check this way, I'll get this updated 👍

Reply:
Updated!